### PR TITLE
Add ability to trigger full opensuse test in trigger-openqa_in_openqa

### PR DESCRIPTION
**Pull request by mimi1vx**
https://progress.opensuse.org/issues/127622

### CODE REVIEW

File: trigger-openqa_in_openqa [hunk 1]
```diff
@@ -46,14 +47,18 @@ trigger() {
...
+    fi
 
     ${client_prefix} "${openqa_cli}" \
         api --host "${target_host_proto}://${target_host}" -X POST isos \
         VERSION=Tumbleweed \
         DISTRI=openQA FLAVOR=dev BUILD="${build}" \
         ARCH="${arch}" BACKEND=qemu WORKER_CLASS=qemu_x86_64 \
         HDD_1="$qcow" \
-        ${ARGS} \
+        "${ARGS}" \
```
Comment (Martchus) [inline trigger-openqa_in_openqa:61]: I don't think quoting will work as the key-value-pairs actually need to be passed as separate arguments. Maybe it makes sense to make `ARGS` a bash array. Then the quoting would make sense.

Comment (mimi1vx) [reply trigger-openqa_in_openqa:63]: why?  it passes and parses the whole cli as a string

Comment (mimi1vx) [reply trigger-openqa_in_openqa:63]: + without quotes is ShellCheck unhappy

Comment (Martchus) [reply trigger-openqa_in_openqa:63]: > without quotes is ShellCheck unhappy

That's true.

> why? it passes and parses the whole cli as a string

If it works then fine :-)

**Comment by Martchus** [comment]
It would be good to add a ticket reference to the commit message.

File: trigger-openqa_in_openqa (hunk 1 above)
Comment (kalikiana) [reply trigger-openqa_in_openqa:63]: How do we know it works? Do we have tests? If we do, why don't they fail since they were not modified in this PR 🤐😉

**Comment by mimi1vx** [comment]
@Martchus you were right ... it doesn't work as excepted ...

**Pull request merged**